tests/presets/test_datapackages.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import division
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from goodtables import presets
+
+
+# Test
+
+def test_datapackages():
+    errors, tables = presets.datapackages([
```

Comment:
Just curious, is this something that is also supported in the other presets (passing the url /path directly vs in `{"source": "'}`)?